test/cli/suggest-kernel/suggest-kernel.out

```diff
@@ -1,6 +1,11 @@
 test/cli/suggest-kernel/suggest-kernel.rb:4: Method `raise` does not exist on `Foo` https://srb.help/7003
      .. |    raise "hi"
             ^^^^^^^^^^
+  Got Foo originating from:
+    test/cli/suggest-kernel/suggest-kernel.rb:3:
+     .. |  def foo
+     .. |    raise "hi"
+     .. |  end
```

Comment:
Looks like for method defs we use the method loc, instead of the declLoc? :psyduck: